### DISCUSSION TITLE
Latex or Math support in markdown

### DISCUSSION THREAD

**Discussion by Yn37git** [seguinleo/Notida#6] [Ideas]
hey can there be Math support like github Markdown generally has, like :  
$$\int_{-\infty}^\infty e^{-x^2} = \frac{\pi}{2}$$
(above formula with usual $$ {....} $$ syntax is not working)

**Comment by seguinleo** [comment]
I'll think about it, but Marked doesn't naturally support math expressions. I have to use an other external library, and that can pose compromises. For now you can use HTML.

**Reply by seguinleo** [reply]
Close this discussion, as Katex has been supported since version 25.4.1

[Guide](https://github.com/seguinleo/Notida/wiki/Markdown)
